### ISSUE TITLE
Jellyfin on Roku TV unable to play mp4 files

### ISSUE THREAD

**Issue by ohshitgorillas** [jellyfin/jellyfin#10515]
All of the files in this folder generate the same error. The only error on the TV app is "An error was encountered while playing this item."

The problem is not with all mp4 files, only this season of this show.

General
```
Complete name                            : \\10.0.0.238\media\tv\The Witcher\The.Witcher.S03.2160p.COMPLETE.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.Multi.Sub.DDP5.1.Atmos.DV.x265.MP4-BEN.THE.MEN\The.Witcher.S03E01.2160p.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/dby1/iso2/mp41)
File size                                : 8.55 GiB
Duration                                 : 1 h 2 min
Overall bit rate mode                    : Variable
Overall bit rate                         : 19.5 Mb/s
Frame rate                               : 24.000 FPS
Movie name                               : The.Witcher.S03E01.2160p.Dolby.Vision[Ben The Men]
Performer                                : BTM
Recorded date                            : 2023
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Writing application                      : Lavf60.5.100
Cover                                    : Yes
Comment                                  : BEN.THE.MEN.TORRENTS

Video
ID                                       : 1
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main 10@L5@Main
HDR format                               : Dolby Vision, Version 1.0, dvhe.08.06, BL+RPU, HDR10 compatible / SMPTE ST 2086, HDR10 compatible
Codec ID                                 : hev1
Codec ID/Info                            : High Efficiency Video Coding
Duration                                 : 1 h 0 min
Bit rate                                 : 18.6 Mb/s
Width                                    : 3 840 pixels
Height                                   : 2 160 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 24.000 FPS
Minimum frame rate                       : 22.890 FPS
Maximum frame rate                       : 24.390 FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0 (Type 2)
Bit depth                                : 10 bits
Bits/(Pixel*Frame)                       : 0.094
Stream size                              : 7.87 GiB (92%)
Language                                 : English
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Color range                              : Limited
Color primaries                          : BT.2020
Transfer characteristics                 : PQ
Matrix coefficients                      : BT.2020 non-constant
Mastering display color primaries        : Display P3
Mastering display luminance              : min: 0.0001 cd/m2, max: 1000 cd/m2
Maximum Content Light Level              : 1053 cd/m2
Maximum Frame-Average Light Level        : 358 cd/m2
Menus                                    : 8
mdhd_Duration                            : 3632083
Codec configuration box                  : hvcC+dvvC

Audio #1
ID                                       : 2
Format                                   : E-AC-3 JOC
Format/Info                              : Enhanced AC-3 with Joint Object Coding
Commercial name                          : Dolby Digital Plus with Dolby Atmos
Codec ID                                 : ec-3
Duration                                 : 1 h 2 min
Bit rate mode                            : Constant
Bit rate                                 : 768 kb/s
Channel(s)                               : 6 channels
Channel layout                           : L R C LFE Ls Rs
Sampling rate                            : 48.0 kHz
Frame rate                               : 31.250 FPS (1536 SPF)
Compression mode                         : Lossy
Stream size                              : 346 MiB (4%)
Language                                 : English
Service kind                             : Complete Main
Default                                  : Yes
Alternate group                          : 1
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Menus                                    : 8
Complexity index                         : Not present / 16
Number of dynamic objects                : 15
Bed channel count                        : 1 channel
Bed channel configuration                : LFE

Audio #2
ID                                       : 3
Format                                   : E-AC-3 JOC
Format/Info                              : Enhanced AC-3 with Joint Object Coding
Commercial name                          : Dolby Digital Plus with Dolby Atmos
Codec ID                                 : ec-3
Duration                                 : 1 h 2 min
Bit rate mode                            : Constant
Bit rate                                 : 768 kb/s
Channel(s)                               : 6 channels
Channel layout                           : L R C LFE Ls Rs
Sampling rate                            : 48.0 kHz
Frame rate                               : 31.250 FPS (1536 SPF)
Compression mode                         : Lossy
Stream size                              : 346 MiB (4%)
Language                                 : Spanish
Service kind                             : Complete Main
Default                                  : No
Alternate group                          : 1
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Menus                                    : 8
Complexity index                         : Not present / 16
Number of dynamic objects                : 15
Bed channel count                        : 1 channel
Bed channel configuration                : LFE

Text #1
ID                                       : 4
Format                                   : Timed Text
Muxing mode                              : sbtl
Codec ID                                 : tx3g
Duration                                 : 56 min 38 s
Bit rate mode                            : Variable
Bit rate                                 : 59 b/s
Frame rate                               : 0.350 FPS
Stream size                              : 24.6 KiB (0%)
Language                                 : English
Default                                  : Yes
Forced                                   : No
Alternate group                          : 3
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Count of events                          : 707
Menus                                    : 8

Text #2
ID                                       : 5
Format                                   : Timed Text
Muxing mode                              : sbtl
Codec ID                                 : tx3g
Duration                                 : 57 min 10 s
Bit rate mode                            : Variable
Bit rate                                 : 69 b/s
Frame rate                               : 0.407 FPS
Stream size                              : 29.0 KiB (0%)
Language                                 : English
Default                                  : No
Forced                                   : No
Alternate group                          : 3
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Count of events                          : 841
Menus                                    : 8

Text #3
ID                                       : 6
Format                                   : Timed Text
Muxing mode                              : sbtl
Codec ID                                 : tx3g
Duration                                 : 56 min 38 s
Bit rate mode                            : Variable
Bit rate                                 : 65 b/s
Frame rate                               : 0.350 FPS
Stream size                              : 26.9 KiB (0%)
Language                                 : Spanish
Default                                  : No
Forced                                   : No
Alternate group                          : 3
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Count of events                          : 707
Menus                                    : 8

Text #4
ID                                       : 7
Format                                   : Timed Text
Muxing mode                              : sbtl
Codec ID                                 : tx3g
Duration                                 : 56 min 38 s
Bit rate mode                            : Variable
Bit rate                                 : 64 b/s
Frame rate                               : 0.350 FPS
Stream size                              : 26.6 KiB (0%)
Language                                 : Portuguese
Default                                  : No
Forced                                   : No
Alternate group                          : 3
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Count of events                          : 707
Menus                                    : 8

Menu #1
ID                                       : 8
Format                                   : Timed Text
Codec ID                                 : text
Duration                                 : 1 h 2 min
Language                                 : English
Encoded date                             : 2023-07-27 20:03:19 UTC
Tagged date                              : 2023-07-27 20:03:19 UTC
Bit rate mode                            : CBR
Menu For                                 : 1,2,3,4,5,6,7
Duration_FirstFrame                      : 3170052
00:00:00.000                             : Part 01
00:57:52.450                             : Credits
Bit rate mode                            : Constant

Menu #2
00:00:00.000                             : Part 01
00:57:52.450                             : Credits
```

**Logs**
```
[2023-11-01 17:23:37.757 -07:00] [INF] [95] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "media". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2023-11-01 17:23:37.758 -07:00] [INF] [95] Jellyfin.Api.Helpers.MediaInfoHelper: RemoteClientBitrateLimit: 35000000, RemoteIp: "10.0.0.27", IsInLocalNetwork: True
[2023-11-01 17:23:37.759 -07:00] [INF] [95] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Anonymous Profile", Path="/tv/The Witcher/The.Witcher.S03.2160p.COMPLETE.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.Multi.Sub.DDP5.1.Atmos.DV.x265.MP4-BEN.THE.MEN/The.Witcher.S03E01.2160p.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.mp4", AudioStreamIndex=1, SubtitleStreamIndex=-1 ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/2df27380-3b33-a491-d3e7-9ed8da1b26ac/stream.mov?MediaSourceId=2df273803b33a491d3e79ed8da1b26ac&Static=true&VideoCodec=hevc&AudioCodec=eac3&AudioStreamIndex=1&api_key=<token>&SubtitleMethod=Encode&Tag=675366a30e39f6d2e8b69d35cc2239bc"
[2023-11-01 17:23:38.483 -07:00] [INF] [101] MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder: "/usr/lib/jellyfin-ffmpeg/ffmpeg" "-i file:\"/tv/The Witcher/The.Witcher.S03.2160p.COMPLETE.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.Multi.Sub.DDP5.1.Atmos.DV.x265.MP4-BEN.THE.MEN/The.Witcher.S03E01.2160p.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.mp4\" -copyts -map 0:3 -an -vn -c:s srt \"/config/data/data/subtitles/1/15c95994-88fe-8ced-a5f4-e6b498586043.srt\""
[2023-11-01 17:23:38.782 -07:00] [INF] [79] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Roku" "1.6.6" playing "Shaerrawedd". Stopped at "0" ms
[2023-11-01 17:23:44.450 -07:00] [INF] [74] MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder: ffmpeg subtitle extraction completed for "file:\"/tv/The Witcher/The.Witcher.S03.2160p.COMPLETE.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.Multi.Sub.DDP5.1.Atmos.DV.x265.MP4-BEN.THE.MEN/The.Witcher.S03E01.2160p.Dolby.Vision.And.HDR10.ENG.And.ESP.LATINO.mp4\"" to "/config/data/data/subtitles/1/15c95994-88fe-8ced-a5f4-e6b498586043.srt"
```

**FFmpeg Logs**
Unfortunately no FFmpeg logs are generated.

**Server System (please complete the following information):**
 - OS: Docker on Ubuntu Server
 - Jellyfin Version: latest (updated daily)
 - Hardware settings & device: NVENC on GTX1660 Ti; Intel i5-10600KF
 - Reverse proxy: nginx
   
**Client System (please complete the following information):**
 - Device: Hisense 65R6+
 - OS: Roku TV 12.5.0
 - Client: Jellyfin app for Roku TV version 1.6 build 6

**Comment by sevenrats** [comment]
Hello. this is the result of directplaying your HDR file.  This issue has already been addressed in unstable, and we could use your help testing the changes before we cut a release. If you are interested in testing the new changes, please follow the "Beta Test" instructions here: https://github.com/jellyfin/jellyfin-roku
If you have any questions feel free to stop by on Matrix or Discord.

Triage team: this affects 1.6.6 but should not affect the following release. Close or retest after roku cuts a later release which contains https://github.com/jellyfin/jellyfin-roku/pull/1299 and related changes

**Comment by linedpaper** [comment]
The dev version fixes the issue for me.

**Comment by ohshitgorillas** [comment]
Hello, resume play now works from the main page of the library (like where it shows "continue watching" and "favorites" etc but it does not work from within the library.

**Comment by Bond-009** [comment]
Should be fixed by jellyfin-roku [v2.0.0](https://github.com/jellyfin/jellyfin-roku/releases/tag/v2.0.0)
